pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/Messages.properties

```diff
@@ -123,7 +123,7 @@ ModelValidatorImpl.InvalidUnnamedParameterType=Expecting "{0}" but got "{1}" of
 ModelValidatorImpl.NoSteps=No steps specified for branch
 ModelValidatorImpl.RequiredSection=Missing required section "{0}"
 ModelValidatorImpl.NoStageName=Stage does not have a name
-ModelValidatorImpl.NothingForStage=No "steps" or "parallel" to execute within stage "{0}"
+ModelValidatorImpl.NothingForStage=No "steps", "stages", or "parallel" to execute within stage "{0}"
```

Comment:
`Expected one of "steps", "stages", or "parallel" for stage "{0}"`

(__No__ _item series_ + __or__ _item_`...` is a mess in English grammar)